pages/05_tags.py

```diff
@@ -30,10 +31,11 @@
         "Number of Tags" : []
         }
 
-    for team in team_list:
+    tag_id = con.sql(f"select id from scouting.allowed_tags where tag_name = '{selected_tag}';").df()['id'].values.tolist()[0]
 
+    for team in team_list:
 
-        tag_count = con.sql(f"SELECT COUNT(*) FROM scouting.tags WHERE team_number = {team} AND tag = '{selected_tag}';").fetchall()[0][0]
+        tag_count = con.sql(f"SELECT COUNT(*) FROM scouting.tags WHERE team_number = {team} AND tag_id = '{tag_id}';").fetchall()[0][0]
```

Comment:
we should not loop over the teams selecting the count.  the rigth way to do this is to do the select once ( select team_number, count(*) from scouting.tags;)
and then pull data out of that df. rule: never write sql in a loop.